pykickstart/commands/firewall.py

```diff
@@ -240,3 +240,23 @@ def __str__(self):
             return retval + "%s\n" % svcstr
         else:
             return retval
+
+class F28_Firewall(F20_Firewall):
+    def __init__(self, writePriority=0, *args, **kwargs):
+        F20_Firewall.__init__(self, writePriority, *args, **kwargs)
+        self.use_system_defaults = kwargs.get("use_system_defaults", None)
+
+    def _getParser(self):
+        op = F20_Firewall._getParser(self)
+        op.add_argument("--use-system-defaults", dest="use_system_defaults",
+                        version=F28, help="""
+                        Don't configure the firewall at all. This instructs anaconda
+                        to do nothing and allows the system to rely on the defaults
+                        that were provided with the package or ostree.""")
+        return op
+
+    def __str__(self):
+        if self.use_system_defaults:
+            return "# Firewall configuration\nfirewall --use-system-defaults\n"
+        else:
+            return F20_Firewall.__str__(self)
```

Comment:
This is going to trip up whoever has to make a new class that inherits from this one and needs to do their own __str__ method, but I don't see that we can do much about that right now.